package/MDAnalysis/coordinates/PQR.py

```diff
@@ -92,10 +92,10 @@
 
    pdb2pqr --ff=charmm --whitespace 4ake.pdb 4ake.pqr
 
-.. _PQR:     http://www.poissonboltzmann.org/file-formats/biomolecular-structurw/pqr
-.. _APBS:    http://www.poissonboltzmann.org/apbs
-.. _PDB2PQR: http://www.poissonboltzmann.org/pdb2pqr
-.. _PDB:     http://www.rcsb.org/pdb/info.html#File_Formats_and_Standards
+.. _PQR:     https://apbs-pdb2pqr.readthedocs.io/en/latest/formats/pqr.html
```

Comment:
According to https://travis-ci.org/MDAnalysis/mdanalysis/jobs/354516290, the target PQR is defined elsewhere, too, and the doc build fails. Please find and fix.

Reply:
I found that and replaced it.

Reply:
Thanks!